docker/release/centos7/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM centos:latest
+
+LABEL maintainer="Jumping Yang <yzpcreate@qq.com>"
+LABEL readme.md="https://github.com/Microsoft/omi/blob/master/docker/README.md"
+LABEL description="This Dockerfile will install the latest release of OMI."
+
+ARG rootpassword=newpassword
+
+SHELL ["/bin/bash", "-c"] 
+RUN echo $rootpassword |passwd root --stdin
+RUN yum update -y; \
+    yum install -y rpm-devel pam-devel openssl which rpm-build krb5-devel redhat-lsb-core;
+RUN rpm -Uvh http://packages.microsoft.com/config/rhel/7/packages-microsoft-prod.rpm
+RUN yum install -y omi
+RUN sed -i 's/httpsport=0/httpsport=0,5986/g' /etc/opt/omi/conf/omiserver.conf
+EXPOSE 5985 5986
+ENTRYPOINT /opt/omi/bin/omiserver --loglevel 4 --httptrace -d && echo 'Started omi inner docker' && tail -f /var/opt/omi/log/omiserver.log
```

Comment:
Why start OMI with `--loglevel 4 --httptrace`? When running released code, isn't this rather wordy if the customer isn't having a problem?

This also relates to other release files too ...